lib/view_model/trade_details_view_model.dart

```diff
@@ -90,6 +91,15 @@ abstract class TradeDetailsViewModelBase with Store {
           value: trade.provider.toString()));
     }
 
+    if (trade.provider.title == "ChangeNOW") {
+      items.add(TrackTradeListItem(
+          title: "Track",
+          value: "https://changenow.io/exchange/txs/${trade.id.toString()}",
+          onTap: () {
+            launch("https://changenow.io/exchange/txs/${trade.id.toString()}");
```

Comment:
Use single quotes instead of double. Don't repeat the link twice, use `final url = https://changenow.io/exchange/txs/${trade.id.toString()};` for not repeat yourself.